sql-plugin/src/main/311until320-nondb/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids.shims
+package org.apache.spark.sql.rapids.shims
```

Comment:
we wouldn't need to move this one, correct?  They only changed in 3.2.1 and later?

Reply:
please file a followup issue